test/unit_test/character_sheet_unit_tests/resources/defaultRaceData.json

```diff
@@ -0,0 +1,130 @@
+[
+	{
+		"BACKGROUND": "ANDER_SURVIVOR",
+		"RACE": "HUMAN"
+	},
+	{
+		"BACKGROUND": "ANTIVAN_WAYFARER",
+		"RACE": "HUMAN"
+	},
+	{
+		"BACKGROUND": "AVVAR",
+		"RACE": "HUMAN"
+	},
+	{
+		"BACKGROUND": "CHASIND_WILDER",
+		"RACE": "HUMAN"
+	},
+	{
+		"BACKGROUND": "CITY_ELF",
+		"RACE": "ELF"
+	},
+	{
+		"BACKGROUND": "DALISH_ELF",
+		"RACE": "ELF"
+	},
+	{
+		"BACKGROUND": "DWARF_DUSTER",
+		"RACE": "DWARF"
+	},
+	{
+		"BACKGROUND": "ELF_APOSTATE",
+		"RACE": "ELF"
+	},
+	{
+		"BACKGROUND": "ELF_CIRCLE_MAGE",
+		"RACE": "ELF"
+	},
+	{
+		"BACKGROUND": "ESCAPED_ELVEN_SLAVE",
+		"RACE": "ELF"
+	},
+	{
+		"BACKGROUND": "FERELDAN_CRAFTSMEN",
+		"RACE": "HUMAN"
+	},
+	{
+		"BACKGROUND": "FERELDAN_FREEMAN",
+		"RACE": "HUMAN"
+	},
+	{
+		"BACKGROUND": "FERELDAN_NOBLE",
+		"RACE": "HUMAN"
+	},
+	{
+		"BACKGROUND": "FREE_MARCHER",
+		"RACE": "HUMAN"
+	},
+	{
+		"BACKGROUND": "HIGH_BORN_DWARF",
+		"RACE": "DWARF"
+	},
+	{
+		"BACKGROUND": "HUMAN_APOSTATE",
+		"RACE": "HUMAN"
+	},
+	{
+		"BACKGROUND": "HUMAN_CIRCLE_MAGE",
+		"RACE": "HUMAN"
+	},
+	{
+		"BACKGROUND": "LOW_BORN_DWARF",
+		"RACE": "DWARF"
+	},
+	{
+		"BACKGROUND": "NEVARRAN_ADVENTURER",
+		"RACE": "HUMAN"
+	},
+	{
+		"BACKGROUND": "ORLESIAN_COMMONER",
+		"RACE": "HUMAN"
+	},
+	{
+		"BACKGROUND": "ORLESIAN_EXILE",
+		"RACE": "HUMAN"
+	},
+	{
+		"BACKGROUND": "ORLESIAN_NOBLE",
+		"RACE": "HUMAN"
+	},
+	{
+		"BACKGROUND": "ORLESIAN_STUDENT",
+		"RACE": "HUMAN"
+	},
+	{
+		"BACKGROUND": "QUNARI_BERESAAD",
+		"RACE": "QUNARI"
+	},
+	{
+		"BACKGROUND": "RIVAINI_MERCHANT",
+		"RACE": "HUMAN"
+	},
+	{
+		"BACKGROUND": "SEHERON_CONVERT",
```

Comment:
Nah, innen fogom indítani a review-t, ugyanis arra a következtetésre jutottam, hogy vannak olyan Backgroundok aminek a faja nem triviális. Ergo nem elég egy "RACE" fieldet tartanod a JSON fileban, egy "RACE_ARRAY" field kell ami egy tömb és tartalmazza a megengedett fajokat. Pl. Seheron convert esetében ez úgy nézne ki, hogy:
```
{
	"BACKGROUND": "SEHERON_CONVERT",
	"RACE_ARRAY":
		[
			"HUMAN",
			"ELF"
		]
},
```

Ahol maradnak a fajok ott nyilván singleton tömbökről fogunk beszélni. Ahol ez még felmerülhet az a Rivaini Merchant, ahol szintén szerintem előfordulhat elf is, illetve a Waking Sea Raider ahol szintén elf is előfordulhat. A dwarfot és a qunarit nem sorolnám ide, mivel azoknak a meglévő backgroundjai jól bemesélhetőek azokba a helyzetekbe is amelyeket ezek megfogalmaznak, és statok szempontjából korrektebbek a fajra nézve.

Reply:
ok